### ISSUE TITLE
issues  ingesting IBM HREA forecast layers

### ISSUE THREAD

**Issue by iferencik** [UNDP-Data/geohub#3767]
### Issue Description

IBM has produced COGs representing HREA forecasts for several years that when ingested had several issues like invalid colors , weird projection, etc. 
I spend 1.5 days trying to process this data (binary thresholding, reprojection to an equal area projection and zonal stats using exact extract) but faced constant issues like empty square areas and weird draw behaviour. 

After a thorough investigation it turns out the data had several issues

1. nan pixels
2. incorrect  metadata
3. not the most suitable data type

Bottom line the original data has to be fixed first by

  removing nan
  changing nodata value
  potentially reprojecting to a different projection than 3857

### References

no links

**Comment by JinIgarashi** [comment]
Closed by https://github.com/UNDP-Data/geohub/pull/4799
